src/pagesComponents/CreateItemAI/index.tsx

```diff
@@ -0,0 +1,188 @@
+            collectionSymbol: '',
+          });
+        }}></CreateNFTSyncChainModal>
+    );
+  };
+
+  const renderUploadComp = () => {
+    const props =
+      createType === 'single'
+        ? {
+            isBatch: false,
+          }
+        : {
+            metaList,
+            isBatch: true,
+          };
+
+    return (
+      <div>
+        <ImagePreview />
+        {collection && createType === 'single' && (
+          <CollectionTag
+            nftName={nftName}
+            collectionName={collection.tokenName}
+            src={collection.logoImage}
+            id={tokenId}
+          />
+        )}
+        {/* <div className=" leading-[24px] w-[343px]  mdl:w-[480px] text-[16px] text-[var(--text-secondary)] mt-[16px]">
+          File types support: JPG,PNG,GIF. Max size: 100MB
+        </div> */}
+      </div>
+    );
+  };
+
+  return (
+    <section className="max-w-[1360px] !mx-auto !px-4 mb-20">
+      <h1
+        className={`flex gap-4 font-semibold text-2xl mdl:text-5xl pt-10 mdTW:pt-12 pb-6 mdTW:pb-10 text-textPrimary !mb-0`}>
+        <span
+          className="flex items-center justify-center w-8 h-8 mdl:w-12 mdl:h-12 rounded-full bg-fillHoverBg cursor-pointer"
+          onClick={onBackHandler}>
+          <BackIcon className=" fill-textPrimary" />
+        </span>
+        {type === 'single' ? 'Create an item' : 'Create Items'}
+      </h1>
+
+      <div
+        className="w-10 h-10"
+        onClick={async () => {
+          setIsLoading(true);
+          const images = await CreateArt();
+          setIsLoading(false);
+          setImages(images!);
+          setAlbumOpen(true);
+        }}>
+        Contract
+      </div>
+
+      <div className="flex gap-x-10">
+        {isSmallScreen ? null : renderUploadComp()}
+        <div className="flex-1 ">
+          <CreateSegmented
+            value={createType}
+            onChange={(value: SegmentedValue) => {
+              onCreateTypeChange(value);
+              setType(value);
+            }}
+          />
+
+          {!isSmallScreen ? null : renderUploadComp()}
+
+          {createType === 'single' ? (
+            <SingleCreateForm optionsForCollection={optionsForCollection} onCreateHandler={onCreateHandler} />
+          ) : (
+            <BatchCreateForm
+              metadataList={metaList}
+              optionsForCollection={optionsForCollection}
+              onCreateHandler={onBatchCreateHandler}
+              onChangeMetaList={setMetaList}
+              createLoading={batchCreateLoading}
+            />
+          )}
+        </div>
+      </div>
+
+      <Progressing isModalOpen={isLoading} />
+      <Album
+        isModalOpen={albumOpen}
+        items={images!}
+        onConfirm={(items: string[]) => {
+          setAlbumOpen(false);
+        }}
+        onCancel={() => {
+          setAlbumOpen(false);
+        }}
+      />
+      {/* <Album
+        isModalOpen={true}
+        onConfirm={(items: string[]) => {
```

Comment:
delete it

Reply:
this file is no longer in use,  has deleted